client/verta/verta/operations/monitoring/summaries.py

```diff
@@ -0,0 +1,205 @@
+    def __init__(self, conn, conf, msg):
+        super(Summary, self).__init__(conn, conf, SummaryService, "summary", msg)
+        self._conn = conn
+        self._conf = conf
+        self.monitored_entity_id = msg.monitored_entity_id
+        self.name = msg.name
+        self.type = msg.type_name
+
+    def __repr__(self):
+        return "Summary name:{}, type:{}, monitored_entity_id:{}".format(self.name, self.type, self.monitored_entity_id)
+
+    def log_sample(self, data, labels, time_window_start, time_window_end, created_at=None):
+        if not created_at:
+            created_at = time_utils.now()
+
+        if isinstance(data, data_types._VertaDataType):
+            content = json.dumps(data._as_dict())
+        else:
+            content = data
+
+        created_at_millis = time_utils.epoch_millis(created_at)
+        window_start_millis = time_utils.epoch_millis(time_window_start)
+        window_end_millis = time_utils.epoch_millis(time_window_end)
+
+        msg = CreateSummarySample(
+            summary_id=self.id,
+            content=content,
+            labels=labels,
+            created_at_millis=created_at_millis,
+            time_window_start_at_millis=window_start_millis,
+            time_window_end_at_millis=window_end_millis,
+        )
+
+        endpoint = "/api/v1/summaries/createSample"
+        response = self._conn.make_proto_request("POST", endpoint, body=msg)
+        result_msg = self._conn.must_proto_response(response, SummarySampleProto)
+        return SummarySample(self._conn, self._conf, result_msg)
+
+    def find_samples(
+        self,
+        sample_ids=None,
+        labels=None,
+        time_window_start=None,
+        time_window_end=None,
+    ):
+        summaries_proto = SummaryQuery(ids=[self.id])._to_proto_request()
+        time_window_start_at_millis = time_utils.epoch_millis(time_window_start) if time_window_start else None
+        time_window_end_at_millis = time_utils.epoch_millis(time_window_end) if time_window_end else None
+        labels_proto = Summary._labels_proto(labels) if labels else None
+        filterSamples = FilterQuerySummarySample(
+            find_summaries=summaries_proto,
+            sample_ids=sample_ids,
+            labels=labels_proto,
+            time_window_start_at_millis=time_window_start_at_millis,
+            time_window_end_at_millis=time_window_end_at_millis,
+        )
+
+        msg = FindSummarySampleRequest(filter=filterSamples)
+        endpoint = "/api/v1/summaries/findSample"
+        response = self._conn.make_proto_request("POST", endpoint, body=msg)
+        success = self._conn.must_proto_response(
+            response, FindSummarySampleRequest.Response
+        )
+        samples = [SummarySample(self._conn, self._conf, record) for record in success.samples]
+        return samples
+
+    @staticmethod
+    def _labels_proto(labels):
+        return {
+            key: LabelFilterQuerySummarySample(label_value=values)
```

Comment:
There's a [`_internal_utils._utils.as_list_of_str()`](https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/_internal_utils/_utils.py#L1430) that can convert `{"key": "value"}` to `{"key": ["value"]}` which I personally would find helpful (otherwise `LabelFilterQuerySummarySample` ends up iterating through `"value"` to produce `['v', 'a', 'l', 'u', 'e']`)
```suggestion
            key: LabelFilterQuerySummarySample(label_value=as_list_of_str(values))
```